src/app/feature/map/search/search.component.ts

```diff
@@ -16,9 +22,21 @@ export class SearchComponent implements OnInit {
   constructor(
     private readonly _mapService: MapService,
     private readonly _mapDataProviderService: MapDataProviderService,
+    private readonly _mapPathService: MapPathService,
     private readonly _translateService: TranslateService
   ) {}
 
+  public readonly selectedUserLocation$ = this._mapPathService.startPoint$.pipe(
```

Comment:
Too many logic in component, main idea is use only `public readonly selectedUserLocation$ = this._mapPathService.startPoint$`